### ISSUE TITLE
Document endian-ness in readme

### ISSUE THREAD

**Issue by mk-pmb** [ahdinosaur/buffer-to-typed-array#3]
Would be nice to see right on npm how to configure it.

**Comment by ahdinosaur** [comment]
at the moment it's not configurable, as far as i understand both buffers and typed arrays will use the underlying native endian. i have no idea what [`endian.js`](https://github.com/ahdinosaur/buffer-to-typed-array/blob/348cd1b3aa5071e33263c34eecccb6f94efd85ac/endian.js) was intended for, but it's not used by [`index.js`](https://github.com/ahdinosaur/buffer-to-typed-array/blob/348cd1b3aa5071e33263c34eecccb6f94efd85ac/index.js)

**Comment by ahdinosaur** [comment]
seems like [data views](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DataView) would be the way to read and write a specific endian, although i'm no expert here sorry :cactus:

**Comment by mk-pmb** [comment]
Then we should at least mention that it's a planned feature but not implemented yet.

**Comment by ahdinosaur** [comment]
is it something you want to implement in some way? as i have no plans to implement this. :smiley_cat:

**Comment by mk-pmb** [comment]
Nope, I'm currently using [binary-pmb](https://www.npmjs.com/package/binary-pmb) instead. :-)
